web/src/components/CreateTestModal/__tests__/CreateTestModal.test.tsx

```diff
@@ -0,0 +1,15 @@
+import {render} from 'test-utils';
```

Comment:
Using the react-testing-library from the `test-util`. It re-export everything from RTL + a custom render with global Providers.